challenges/easy/who-likes-it/solutions/aleattene/main.go

```diff
@@ -0,0 +1,35 @@
+
+// GO solution for challenge: "Who likes this"
+
+package main
+
+import ("fmt"; "strconv")
+
+func likes(followers ...string) string {
+  if len(followers) == 0 {
```

Comment:
personal taste, what about moving this to switch/case?

Reply:
Advice accepted.
Updated code.